doc/org.eclipse.cdt.doc.user/src/getting_started/cbs_using_existing_code.adoc

```diff
@@ -0,0 +1,66 @@
+////
+Copyright (c) 2000, 2025 Contributors to the Eclipse Foundation
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License 2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-2.0/
+
+SPDX-License-Identifier: EPL-2.0
+////
+
+// pull in shared headers, footers, etc
+:docinfo: shared
+
+// support image rendering and table of contents within GitHub
+ifdef::env-github[]
+:imagesdir: ../../images
+:toc:
+:toc-placement!:
+endif::[]
+
+// enable support for button, menu and keyboard macros
+:experimental:
+
+// Until ENDOFHEADER the content must match adoc-headers.txt for consistency,
+// this is checked by the build in do_generate_asciidoc.sh, which also ensures
+// that the checked in html is up to date.
+// do_generate_asciidoc.sh can also be used to apply this header to all the
+// adoc files.
+// ENDOFHEADER
+
+== Using existing code
+
+This tutorial describes how to create a new Core Build System
+(CBS) project for use with existing code.
+
+Using existing code is done by creating a new **empty** project on top of the
+existing code.
+
+=== Using an existing CMake project
+
+To create an empty CMake project in CDT select the template *Empty or Existing CMake
+Project* in the *New C/{cpp} Project* wizard.
+
+image:cbs_empty_cmake_project.png[Empty or Existing CMake Project]
+
+In the *New CMake Project* wizard deselect *Use default location*
```

Comment:
[Consider replacing "deselect" with "uncheck". Feels a little more user friendly. There are a couple more occurrences of this below]

Reply:
Done